packages/ag-charts-community/src/scale/logScale.ts

```diff
@@ -103,11 +103,12 @@ export class LogScale extends ContinuousScale<number> {
     }
 
     ticks() {
+        this.refresh();
```

Comment:
Is this ordering change consistent with the other scales? (Looks like the others were consistently doing this after the `this.domain` check before)?